internal/dependency/manager/manager_test.go

```diff
@@ -0,0 +1,288 @@
+package manager
+
+import (
+	"github.com/g2a-com/klio/internal/context"
+	"github.com/g2a-com/klio/internal/dependency"
+	"github.com/spf13/afero"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+)
+
+//==========================================================================================
+//::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+//==========================================================================================
+// Take a look at DEP INDEX in utils_test.go
+//==========================================================================================
+
+func TestGetInstalledCommands(t *testing.T) {
+	tests := []struct {
+		// Name of the test
+		name string
+		// Injected mock function for returning local dependencies
+		fetchIndexFunction func(filePath string) (*dependency.DependenciesIndex, error)
+		// Configuration paths for klio
+		paths context.Paths
+		// Desired index list of dependencies installed
+		want []dependency.DependenciesIndexEntry
+	}{
+		{
+			name:               "errorFromIndex",
```

Comment:
Test cases should have meaningful names. Name should describe in non-programmer friendly manner what is an intended behaviour which test case checks.

For more details, please follow those guidelines: 
https://enterprisecraftsmanship.com/posts/you-naming-tests-wrong/

Reply:
Again, putting a link tells me nothing. Please explain. You need the name to be more explicit? Is that it? function+input desc+output desc?